src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -364,31 +365,32 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
         .map(assembly => resourceLoader.loadResource(assembly, `_framework/${assembly}`, lazyAssemblies[assembly], 'assembly'))
         .map(async resource => (await resource.response).arrayBuffer()));
 
+
       return BINDING.js_to_mono_obj(
         Promise.all([resourcePromises, pdbPromises]).then(values => {
-          const resourcesToLoad = values[0];
-          const pdbsToLoad = values[1];
-          if (resourcesToLoad.length) {
+          lazyResources["assemblies"] = values[0];
+          lazyResources["pdbs"] = values[1];
+          if (lazyResources["assemblies"].length) {
             window['Blazor']._internal.readLazyAssemblies = () => {
-              const assemblyBytes = BINDING.mono_obj_array_new(resourcesToLoad.length);
-              for (let i = 0; i < resourcesToLoad.length; i++) {
-                const assembly = resourcesToLoad[i] as ArrayBuffer;
+              const assemblyBytes = BINDING.mono_obj_array_new(lazyResources["assemblies"].length);
+              for (let i = 0; i < lazyResources["assemblies"].length; i++) {
+                const assembly = lazyResources["assemblies"][i] as ArrayBuffer;
                 BINDING.mono_obj_array_set(assemblyBytes, i, BINDING.js_typed_array_to_array(new Uint8Array(assembly)));
               }
               return assemblyBytes;
             };
 
             window['Blazor']._internal.readLazyPdbs = () => {
-              const pdbBytes = BINDING.mono_obj_array_new(resourcesToLoad.length);
-              for (let i = 0; i < resourcesToLoad.length; i++) {
-                const pdb = pdbsToLoad && pdbsToLoad[i] ? new Uint8Array(pdbsToLoad[i] as ArrayBufferLike) : new Uint8Array();
+              const pdbBytes = BINDING.mono_obj_array_new(lazyResources["assemblies"].length);
```

Comment:
Are we able to dereference these inside the function: e.g. `const assemblies = lazyResources["assemblies"];`?

Reply:
Good call! Just updated to reflect this.